heron/schedulers/src/java/com/twitter/heron/scheduler/yarn/HeronExecutorTask.java

```diff
@@ -133,6 +133,7 @@ public HeronExecutorTask(final REEFFileNames fileNames,
         env,
         topologyJar,
         topologyDefFile,
+        false,
```

Comment:
Could you please elaborate what is the difficulty for YARN? A number of configurations are passed to the YARN scheduler. I think it is feasible to pass verbose flag too.